1-basics.md

```diff
@@ -42,13 +43,18 @@ How to navigate VS Code’s user interface.
   - Search
   - Help: `?`
 - Status bar
+- Panel
 - [Accessibility](https://code.visualstudio.com/docs/editor/accessibility) (color themes, screen readers, screencast mode, etc.)
 - [VS Code terminal](https://code.visualstudio.com/docs/terminal/advanced) (xtermjs)
 - [Navigating without a mouse](https://www.youtube.com/watch?v=dJWJ0hCAkAI)
 
 > **Exercise**: Explore VS Code acessibility
 >
-> Find a repo on GitHub and try opening it either in your browser (press `.` when on repo page). Try navigating around the UI, and turn on some acessability settings and give them a go!
+> 1. Browse to the tutorial repo on GitHub and try opening it either in your browser (press `.` when on repo page) or cloning it locally and opening with `code .`.
+> 2. Use the Command Palette to Zoom in and Zoom out (editor and/or view).
+> 3. Search for Color Themes using the Extensions tab. You can find extensions or color themes that are accessible for color vision deficiencies.
```

Comment:
let's explicitly mention some accessibility settings to try here, like colorblind theme, screen reader, etc.

Reply:
Done!